setup.cfg

```diff
@@ -6,3 +6,20 @@ license_file=LICENSE
 
 [devpi:upload]
 formats=sdist.tgz,bdist_wheel
+
+[mypy]
+check_untyped_defs = True
+ignore_missing_imports = True
+no_implicit_optional = True
+strict_equality = True
+warn_redundant_casts = True
+warn_return_any = True
+warn_unused_configs = True
+warn_unused_ignores = True
+[mypy-pluggy.*]
+disallow_any_decorated = True
+disallow_any_generics = True
+disallow_any_unimported = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
```

Comment:
I find most of the `any` settings more annoying than they are helpful -- they also tend to lead to lower quality annotations in my experience 🤷‍♂

Reply:
Even after everything is annotated?